modules/ROOT/pages/index.adoc

```diff
@@ -185,6 +185,32 @@ Anypoint Monitoring is the standard method of monitoring Mule application and AP
 
 |===
 
+== Use Cases
+
+The following use cases provide examples of how you can use Anypoint Monitoring.
+
+* Understand performance-related KPIs+
+DevOps engineer Gary maintains a series of APIs related to a transaction processing system. Because the TPS requires swift response time, *Gary uses API response time dashboards and alerts to send an email when there is a bottleneck*. After gathering some data, Gary can then size his applications.
+
+* Plan resource allocation across the application network +
+C4E specialist Peyton uses Anypoint Visualizer to view their application network. For CloudHub and Anypoint Runtime Fabric applications, Peyton sees CPU and memory utilization and idle capacity using integrated Anypoint Monitoring.
+
+* Pinpoint root causes of issues +
+Support team member Darius receives a call that the system is running slowly. He examines dashboards and checks response times, failed requests, and so on. He identifies that the backend is not scaling for the number of requests; for Mule apps, he can add capacity, or for other backend issues, he can throttle requests.
```

Comment:
the last part of this sentence confused me on the first reading, could it be reworded to avoid some commas? something like:
**he can add capacity for Mule apps, or he can throttle requests to address other backend issues.**

Reply:
```suggestion
Support team member Darius receives a call that the system is running slowly. He examines dashboards and checks response times, failed requests, and so on. He identifies that the backend is not scaling for the number of requests; he can add capacity for Mule apps, or he can throttle requests for other backend issues.
```